third_party/2and3/toml.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Any, IO, Mapping, MutableMapping, Optional, Protocol, Text, Type, Union
+import datetime
+import sys
+
+if sys.version_info >= (3, 4):
+    import pathlib
+    if sys.version_info >= (3, 6):
+        import os
+        _PathLike = Union[Text, pathlib.PurePath, os.PathLike]
+    else:
+        _PathLike = Union[Text, pathlib.PurePath]
+else:
+    _PathLike = Text
+
+class _Writable(Protocol):
+    def write(self, obj: str) -> Any: ...
+
+class TomlDecodeError(Exception): ...
+
+class TomlTz(datetime.tzinfo):
```

Comment:
Hm, this actually doesn't exist in master (https://github.com/uiri/toml/blob/master/toml/__init__.py), at least not at the top level.

I was looking at my installed version of toml before, which is apparently pretty out of date.

Reply:
Relative to master we're also missing TomlArraySeparatorEncoder and a couple of other Encoder and Decoder classes.

Reply:
I've deleted TomlTz, but I'd prefer not to add the other things -- they're not mentioned in the API reference in the README.rst. Let someone else finish this once they're documented. (The PR that prompted me to do this isn't even going to be merged.)